### DISCUSSION TITLE
Please tell me which versions of Safari are officially supported.

### DISCUSSION THREAD

**Discussion by Taikono-Himazin** [microsoft/monaco-editor#4283] [Q&A] [answered]
There was a report that it cannot be displayed on safari13 (MacOS HighSierra).
Safari 13 is quite old (more than 5 years ago), so I think it's fine if it can't be displayed, but I don't know which version of Safari will work properly.
Is it written somewhere?

**Comment by Taikono-Himazin** [comment]
Please also tell us the supported versions of FireFox, Google Chrome, and Microsoft Edge.

**Comment by hediet** [comment, marked as answer]
We only support the latest browser versions.
